### PR TITLE
Refactoring/cleaner database schemas

### PR DESCRIPTION
**Pull request by stefanheyder**
This PR will rename the old places table to the sections table. Before merging this, we should test the new migrations with an existing DB dump to ensure this does work in production.

While doing this I also added a seed for the bd-Club to the new sections table, so we can test whether the coming ILSCeV integration. Currently it works for statistics and month view.

### CODE REVIEW

**Comment by stefanheyder** [comment]
[[2] (ILSC) Generalize single event view](https://trello.com/c/1YpkyUDB/303-2-ilsc-generalize-single-event-view)

File: app/Schedule.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ public function shifts() {
 	public function getTemplateEntries()
```
Comment (4D44H) [inline app/Schedule.php]: Could be "shifts", but that's no big deal :)

**Comment by 4D44H** [comment]
Used prod DB on my local setup, migrations worked, can't see any errors - seems to work with real data.
Should I wait until the meeting or merge now?

**Comment by stefanheyder** [comment]
@4D44H If you spotted no errors, I vote for merging.

**Pull request merged**